tests/AutoDirectionProvider_test.jsx

```diff
@@ -9,43 +9,44 @@ import mockBrcast from './mocks/brcast_mock';
 
 describe('<AutoDirectionProvider>', () => {
   it('renders a DirectionProvider', () => {
-    const wrapper = shallow((
+    const wrapper = shallow(
       <AutoDirectionProvider text="a">
         <div />
-      </AutoDirectionProvider>
-    )).dive();
+      </AutoDirectionProvider>,
```

Comment:
please double-wrap in parens instead of using the commas - ie, please revert whatever autoformatting was applied.

Reply:
Reverted auto formatting on all files.